README.md

```diff
@@ -5,11 +5,11 @@
 
 ## Overview
 
-This repository contains configuration files and [overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) to deploy Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/).
+Kubeflow on AWS is an open source distribution of [Kubeflow](https://www.kubeflow.org/) that provides its own Kubeflow manifests to support integrations with various AWS managed services. Use Kubeflow on AWS to streamline data science tasks and build highly reliable, secure, and scalable machine learning systems with reduced operational overheads.
```

Comment:
`its own Kubeflow manifests`

- change this to `where we provide our own manifests`
- or just delete it

Reply:
avoided the work we/you to make it more formal similar to website description. Reviewed it with Judy and Sree. lgt both

Reply:
oh okay